src/port1.0/portconfigure.tcl

```diff
@@ -260,6 +295,25 @@ default compiler.fallback       {[portconfigure::get_compiler_fallback]}
 default compiler.blacklist      {}
 default compiler.whitelist      {}
 
+# Compiler Restrictions
+#   compiler.c_standard            Standard for the C programming language (1989, 1999, 2011, etc.)
+#   compiler.cxx_standard          Standard for the C++ programming language (1998, 2011, 2014, 2017, etc.)
+#   compiler.openmp_version        Version of OpenMP required (blank, 2.5, 3.0, 3.1, 4.0, 4.5, etc.)
+#   compiler.mpi                   MacPorts port that provides MPI (blank, mpich, openmpi)
+#   compiler.thread_local_storage  Is thread local storage required, e.g. __thread, _Thread_local, std::thread_local (yes, no)
+options                            \
+    compiler.c_standard            \
+    compiler.cxx_standard          \
+    compiler.openmp_version        \
+    compiler.mpi                   \
+    compiler.thread_local_storage
+
+default compiler.c_standard            1989
```

Comment:
IMO, this should be 1999 as that has been the default in clang for many years now.  I'd love to see it become C11, but I don't think that would be appropriate as part of this change.

Reply:
This change is made in bf864905e12b8489c7be8d1413d0bd5b9d82761c.